### DISCUSSION TITLE
Value mapping infinity

### DISCUSSION THREAD

**Discussion by Betula-L** [grafana/grafana#37391] [General]
**What's requested?**
Add value mapping for Infinity.
1. Range to Text
Current implementation needs both lower and upper limit, otherwise it will be set as 0. However, sometimes we need map value [a, +infinity) or (-infinity, b]. See also https://github.com/grafana/grafana/issues/1319#issuecomment-591390271
2. Special value
Map infinity and -infinity value to special text, rather than 'Infinity'.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
